tests/tools/nnpackage_run/src/args.cc

```diff
@@ -54,6 +54,44 @@ std::unordered_map<uint32_t, Json::Value> argArrayToMap(const Json::Value &jsonv
   return ret;
 }
 
+// param shape_str is a form of, e.g., "[1, [2, 3], 3, []]"
+void handleShapeParam(nnpkg_run::TensorShapeMap &shape_map, const std::string &shape_str)
+{
+  Json::Value root;
+  Json::Reader reader;
+  if (!reader.parse(shape_str, root, false))
+  {
+    std::cerr << "Invalid JSON format for output_sizes \"" << shape_str << "\"\n";
+    exit(1);
+  }
+
+  auto arg_map = argArrayToMap(root);
```

Comment:
@hyunsik-yoon `argArrayToMap` returns `std::unordered_map<uint32_t, Json::Value>`, and `TensorShapeMap = std::unordered_map<int, std::vector<int>>`. What happens if `key` is bigger than int32 max? Perhaps we may change the return type of argArrayToMap to <int32_t, ...> since it is only used for setting `output_size`.

Reply:
@glistening, Fixed `TensorShapeMap` to use key as `uint32_t`. :-)